app/_locales/en/messages.json

```diff
@@ -2427,6 +2430,9 @@
   "inYourSettings": {
     "message": "in your Settings"
   },
+  "included": {
```

Comment:
I checked with the MetaMask team and for generic terms like this it may be best to leave them unscoped to a particular feature so it's easier to find and reuse in other places.